### PR TITLE
chore: fix `go.work` file and run `make format`

### PR DESCRIPTION
**Pull request by Pantani**

### CODE REVIEW

**Review by julienrbrt — CHANGES_REQUESTED** [review]
The go.work and go.work.sum shouldn't be committed.

File: go.work
Comment (jeronimoalbi) [inline go.work:1]: Why do we want to commit `go.work` instead of `go.work.example`?

**Comment by Pantani** [comment]
We agree to not commit the `go.work.sum`. But why do we not commit to `go.work`?

**Comment by julienrbrt** [comment]
> @jeronimoalbi , @julienrbrt . We agree to not commit the `go.work.sum`. But why do we not commit to `go.work`?
> 
> 
> 
> And why keep the `go.work.example` if we are not using it?

go.work makes it easy for local development but it's going to make unclear what versions uses a program when tagging it (as the tagged version should depend on tagged version - like you should tag something that depends on local replaces)

Keeping a go.work.example is handy for anyone using go.work, but when they submit a PR to the repo they need to make sure they only use released version, or split the PR.

There's a lengthy discussion about this in the SDK somewhere.

**Comment by Ehsan-saradar** [comment]
Yep, @julienrbrt is right. This is also the case with [ibc-go](https://github.com/cosmos/ibc-go) repository.

**Pull request closed without merging**